test/test_parse_psse.jl

```diff
@@ -353,8 +353,10 @@ end
     @test IS.compare_values(original_sys, deserialized_sys)
 end
 
+#TODO - add system to PSB to make this test functional
+#=
 @testset "PSSE isolated bus handling (unavailable vs topologically isolated)" begin
-    #sys = build_system(PSSEParsingTestSystems, "isoltaed_bus_test_system"; force_build = true) #TODO - add system to PSB to make this test functional
+    #sys = build_system(PSSEParsingTestSystems, "isolated_bus_test_system"; force_build = true) 
```

Comment:
@luke-kiernan can you please open a PR to add this new system to PSB and then uncomment this test? The raw file is added to PowerSystemsTestData here: 
https://github.com/NREL-Sienna/PowerSystemsTestData/pull/78